ArduCopter/config.h

```diff
@@ -764,3 +764,7 @@
 #ifndef HAL_FRAME_TYPE_DEFAULT
 #define HAL_FRAME_TYPE_DEFAULT AP_Motors::MOTOR_FRAME_TYPE_X
 #endif
+
+#ifndef SMARTAUDIO_ENABLED
+ #define SMARTAUDIO_ENABLED ENABLED
```

Comment:
It probably should not be enabled if HAL_MINIMIZE_FEATURES defined.

Reply:
fixed